### PR TITLE
Removal of Stack Exchange sites that aren't centrally owned

### PR DESCRIPTION
**Pull request by adamrusted**
PR serves to remove all non-centrally owned Stack Exchange sites, following discussion below.

~This PR seeks to include specific guidance when adding Stack Exchange sites to the project, using guidelines proposed by @service-paradis in https://github.com/simple-icons/simple-icons/issues/9225#issuecomment-1667805006.~

~Happy for @simple-icons/maintainers to have a discussion around the threshold, but given we have 2 outstanding icons (Code Golf and Ask Different) I felt we should set some kind of precedent.~

### CODE REVIEW

**Comment by adamrusted** [comment]
> I would say NO to this metric.

Given I'd want to keep whatever metric we use quite simple, do we instead veer toward not accepting Stack Exchange communities at all, and removing the ones we do have?

As you say, a lot are more topic forums as opposed to outright brands themselves (I'm comparing this to adding specific subreddit branding to the repo).

**Comment by LitoMore** [comment]
It's kind of like submitting these icons below to our project:

image

I think Stack Exchange only owns these icons: https://stackoverflow.design/brand/logo/#community-logos

**Comment by LitoMore** [comment]
We can also keep Server Fault and Super User. These two icons are in the guidelines as well.

**Comment by adamrusted** [comment]
No worries, I'll adjust this PR to be a removal of any other Stack Exchange sites we've already added.
For reference:

- [ ] Ask Ubuntu

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ We welcome icon requests. Before you submit a new issue please make sure the ico
     - For websites, the [Similarweb global rank](https://www.similarweb.com) should be less than 500k.
         - Note that for brands that have already been added the threshold for continued inclusion rises to 750k.
     - For GitHub projects, the amount of "stars" should be above 5k.
+    - For Stack Exchange sites, the [visits per day](https://stackexchange.com/sites?view=list#traffic) should be above 50k.
```
Comment (LitoMore) [inline CONTRIBUTING.md:50]: This line can be removed now.

**Review by LitoMore — APPROVED** [review]

**Comment by adamrusted** [comment]
@LitoMore - do we also want a line adding to CONTRIBUTING.md regarding not adding these?

**Comment by LitoMore** [comment]
@adamrusted I think our contributing guide already introduced these kinds of icons that are out of scope. These icons do not belong to Stack Overflow / Stack Exchange.

**Review by mondeja — APPROVED** [review]

**Review by service-paradis — APPROVED** [review]

**Pull request merged**